### DISCUSSION TITLE
Mute all team notifications in a given organization

### DISCUSSION THREAD

**Discussion by buremba** [community/community#147309] [Enterprise]
### Select Topic Area

Question

### Body

 I'm part of some teams in different organisations that belong to multiple companies. I got access mainly to see some repos inside these organizations as they're only available to the teams. Unfortunately the companies I work with don't just username to give access to repos as it's usually a company policy so I have to part of these teams.

However, I started getting many PR review notifications with mention to these teams and I'm not able to keep up with them. I would rather not see any notification unless somebody mentions me explicitly. Is this feasible?

**Comment by buremba** [comment]
Relevant: https://github.com/orgs/community/discussions/23703

**Comment by queenofcorgis** [comment]
Hello @buremba and thank you for your patience. 

There is a toggle in the notification settings to automatically watch new team notifications when added to a team. I am not sure if turning this one off will actually clear existing subscriptions. But at least it would prevent this from happening if you are added to new teams in the future. I'll write back here if I uncover any other information.

Happy New Year!
